sdk/src/main/java/com/silanis/esl/api/model/AccessibleAccountResponse.java

```diff
@@ -0,0 +1,41 @@
+package com.silanis.esl.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.silanis.esl.api.util.SchemaSanitizer;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AccessibleAccountResponse extends Entity implements java.io.Serializable{
+
+    @JsonIgnore
+    public static final String FIELD_ACCOUNT_UID = "accountUid";
+    @JsonIgnore
+    public static final String FIELD_ACCOUNT_NAME = "accountName";
+
+    private String _accountUid;
+    private String _accountName;
```

Comment:
Why do we use _ in front of fields ? I think with proper Jackson annotations we should be able to skip manual validation and fields hiding

Reply:
Or is it because the same object is being used a JSON AND db mapping ?

Reply:
I just follow same naming convention, I think it's nothing for Jackson.